.github/workflows/multibuild.yaml

```diff
@@ -85,8 +91,21 @@ jobs:
           dart compile exe bin/activate_cli.dart -v -o sshnp/at_activate${{ matrix.ext }}
           dart compile exe bin/sshnp.dart -v -o sshnp/sshnp${{ matrix.ext }}
           dart compile exe bin/npt.dart -v -o sshnp/npt${{ matrix.ext }}
+          dart compile exe bin/npp_file.dart -v -o sshnp/npa_file${{ matrix.ext }}
+          dart compile exe bin/npp_file.dart -v -o sshnp/npp_file${{ matrix.ext }}
```

Comment:
standardizing on npp_ prefix for all policy binaries; given we had npa_file out there already, preserving it for now